django_admin_bootstrapped/templates/registration/password_change_form.html

```diff
@@ -1,5 +1,6 @@
 {% extends "admin/base_site.html" %}
 {% load i18n static bootstrapped_goodies_tags %}
+{% load url from future %}
```

Comment:
BTW I think that the `master` branch has updated a little since the last push to PyPI, so I think these lines were removed since this isn't needed in DJ3x.